demos/vector_graphic/lv_demo_vector_graphic.c

```diff
@@ -92,20 +90,18 @@ static void draw_radial_gradient(lv_vector_dsc_t * ctx, lv_vector_path_t * path)
     lv_vector_path_line_to(path, &pts[3]);
     lv_vector_path_close(path);
 
-    lv_grad_dsc_t grad;
-    grad.dir = LV_GRAD_DIR_HOR;
-    grad.stops_count = 2;
-    grad.stops[0].color = lv_color_hex(0xff0000);
-    grad.stops[0].opa = LV_OPA_COVER;
-    grad.stops[0].frac = 0;
-    grad.stops[1].color = lv_color_hex(0x0000ff);
-    grad.stops[1].opa = LV_OPA_COVER;
-    grad.stops[1].frac = 255;
-    // grad.stops[2].color = lv_color_hex(0x0000ff);
-    // grad.stops[2].opa = LV_OPA_COVER;
-    // grad.stops[2].frac = 255;
-
-    lv_vector_dsc_set_fill_radial_gradient(ctx, &grad, 50, 50, 20, LV_VECTOR_GRADIENT_SPREAD_REFLECT);
+    lv_gradient_stop_t stops[2];
+    lv_memzero(stops, sizeof(stops));
+    stops[0].color = lv_color_hex(0xff0000);
+    stops[0].opa = LV_OPA_COVER;
+    stops[0].frac = 0;
+    stops[1].color = lv_color_hex(0x0000ff);
+    stops[1].opa = LV_OPA_COVER;
+    stops[1].frac = 255;
+
+    lv_vector_dsc_set_fill_radial_gradient(ctx, 450, 100, 20);
```

Comment:
The radial gradient `cx` and `cy` are absolute coordinates now?

Reply:
Yes, the rendering behavior of both ThorVG and VG-Lite is absolute coordinates relative to the draw buffer.